specification/attributes/column_naming_and_ordering_convention.md

```diff
@@ -1,23 +1,24 @@
-# Column Naming Convention
+# Column Naming and Ordering
```

Comment:
Can you also update the changelog in the root to add this rename as a subbullet under Changed > Column naming convention?